include/rmm/rmm_api.h

```diff
@@ -53,6 +53,8 @@ typedef enum
   CudaDefaultAllocation = 0,  //< Use cudaMalloc for allocation
   PoolAllocation        = 1,  //< Use pool suballocation strategy
   CudaManagedMemory     = 2,  //< Use cudaMallocManaged rather than cudaMalloc
+  ManagedPoolAllocation = 3,  //< Use pool suballocation strategy with
```

Comment:
Please don't add this option, it already exists. Specify a managed pool by using `PoolAllocation | CudaManagedMemory`, which just happens to equal `3`...